kornia/contrib/models/efficient_vit/nn/ops.py

```diff
@@ -8,7 +8,17 @@
 import torch
 import torch.nn.functional as F
 from torch import nn
-from torch.cuda.amp import autocast
+
+from kornia.utils._compat import torch_version_ge
+
+if torch_version_ge(2, 4):
+    from functools import partial
+
+    from torch.amp import autocast as _autocast
+
+    autocast = partial(_autocast, "cuda")
+else:
+    from torch.cuda.amp import autocast
```

Comment:
maybe we should do these imports under the `kornia.utils._compat`, and import it from there? cc @edgarriba